swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
```

Comment:
small tweak, but I believe we could shorten this to:

```suggestion
from .item import Item
```